easybuild/easyconfigs/l/lpsolve/lpsolve-5.5.2.5-GCC-8.3.0.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'CmdCp'
+
+name = 'lpsolve'
+version = '5.5.2.5'
+
+homepage = 'https://sourceforge.net/projects/lpsolve/'
+description = "Mixed Integer Linear Programming (MILP) solver"
+
+toolchain = {'name': 'GCC', 'version': '8.3.0'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['lp_solve_%(version)s_source.tar.gz']
+patches = ['lpsolve-%(version)s_Fix_expected_identifier_error_for_gcc.patch']
+checksums = [
+    '201a7c62b8b3360c884ee2a73ed7667e5716fc1e809755053b398c2f5b0cf28a',  # lp_solve_5.5.2.5_source.tar.gz
+    # lpsolve-5.5.2.5_Fix_expected_identifier_error_for_gcc.patch
+    'be46d06035797b455ff616c74f83c4e36c9cad6c41b0c8680b29cb6b3208f49c',
+]
+
+dependencies = [('binutils', '2.32')]
```

Comment:
@lexming binutils is already a dependency of GCC, so you can remove it.

Reply:
I didn't realize that `GCC/8.3.0` also includes `binutils` :open_mouth: 
Fixed :+1: